### PR TITLE
V0.1_Its_Not_a_Sexbolt_switch

### PR DESCRIPTION
**Pull request by Polar-Ted**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folder and file naming:
    * Folders and filenames shouldn't contain spaces. Only letters `a-zA-Z`, numbers `0-9`, underscores `_`, hyphens `-` and periods `.`
    * Primary color: `part_xyz.stl`
    * Accent color: `[a]_part_xyz.stl`
    * Opaque color (Blocks light): `[o]_part_xyz.stl`
    * Clear/transparent color (Allows light): `[c]_part_xyz.stl`
    * Quantity, if more than one is needed: `part_xyz_x4.stl`
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
Sexbolt style switch for the V0 using a fender washer as a larger hat for the nozzle to reach on a V0
#### How was it tested? 
Printed and used for the last month on V0.1781

#### Any background context you want to provide?

Designed to bring Auto Z offset to the V0

#### Screenshots (if appropriate)
https://github.com/Polar-Ted/STL-Files/blob/main/V0.1_Its_Not_a_Sexbolt_switch/Images/NotaSexboltimage.png

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -308,6 +308,7 @@ like so:
 |piet206 |[V1.8 Z Rails Mod](./piet206/V1.8._Z_Rails_Mod) | Add MGN12H linear rails for Z axis motion |:x: |:heavy_check_mark: |:x: |:x:|
 | Polared | [Power Inlet with Keystone Insert](./Polared/Power_Inlet_w_Keystone_Insert) | Voron 0.1 Power inlet with keystone insert slot | :heavy_check_mark: | :x: | :x: | :x: | :x:|
 |Polar-Ted|[V0.1 Two Piece Back Panel](./Polar-Ted/V0_Two_Piece_Back_Panel/) | Vented 2 piece back panel for V0 |:heavy_check_mark: |:x: |:x: |:x:|
+| |[V0.1 Z nozzle endstop switch](./Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/) | Sexbolt style switch for V0 |:heavy_check_mark: |:x: |:x: |:x:|
```
Comment (newphreak) [inline printer_mods/README.md:311]: Can you add one more :x: at the end of both your mods? Just to fill out the whole table :)

**Review by newphreak — CHANGES_REQUESTED** [review]
Looks good to me. Just missing the trident cross in the table.

File: printer_mods/README.md (hunk 1 above)
Comment (Polar-Ted) [reply printer_mods/README.md]: done

**Review by newphreak — APPROVED** [review]
Looks good to me

File: printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+[Video of probing action](https://youtu.be/bdxmqTDDsMM)
+
+**BOM**
+
+ 1 x HeartK Voron 2.4 Endstop switch      
+ 4 x M2x8 Self Tapping screws      
+ 3 x 5x7x8 Sleeve Bearing      
+ 1 x M3 x 20mm round amuminum standoff post ( 5MM od)   
+      [Example of what to look for](https://www.aliexpress.com/item/2251832695751106.html?spm=a2g0o.productlist.0.0.39d572d1Ikaegt&algo_pvid=0ef968d6-c41a-4bf9-8728-aab261ae0f26&aem_p4p_detail=202207041141012168556211044480003127304&algo_exp_id=0ef968d6-c41a-4bf9-8728-aab261ae0f26-4&pdp_ext_f=%7B%22sku_id%22%3A%2265705928134%22%7D&pdp_npi=2%40dis%21USD%21%212.23%21%21%21%21%21%402132f35216569600607318207ec8b9%2165705928134%21sea). I did not buy from this store.. Just using them as a reference for the type of post you need.
```
Comment (cbc02009) [inline printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD:11]: please delete the `?` and everything after it in the link, it's all tracking info.

Comment (cbc02009) [reply printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD:11]: so just `https://www.aliexpress.com/item/2251832695751106.html`

File: printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+ 1 x M3 x 20mm round amuminum standoff post ( 5MM od)   
+      [Example of what to look for](https://www.aliexpress.com/item/2251832695751106.html?spm=a2g0o.productlist.0.0.39d572d1Ikaegt&algo_pvid=0ef968d6-c41a-4bf9-8728-aab261ae0f26&aem_p4p_detail=202207041141012168556211044480003127304&algo_exp_id=0ef968d6-c41a-4bf9-8728-aab261ae0f26-4&pdp_ext_f=%7B%22sku_id%22%3A%2265705928134%22%7D&pdp_npi=2%40dis%21USD%21%212.23%21%21%21%21%21%402132f35216569600607318207ec8b9%2165705928134%21sea). I did not buy from this store.. Just using them as a reference for the type of post you need.    
+ 2 x M3 x 8mm Flat CAP screw      
+ 1 X M5 Stainless Fender Washer 15mm OD      
+ 2 x M3 standard washer      
+ 2x M3 x 8mm or 10mm button head screws      
+ 2x M3 square or hex nuts      
+ 220 and 400 grit sandpaper for deburing the fender washer and sizing the standoff post if needed.      
+ 
+ I like the Deepfriedhero [endstop super kit for the switch and bearings](https://deepfriedhero.in/products/voron-2-4-sexbolt-z-endstop-super-kit?_pos=4&_sid=4148419ee&_ss=r) if you are in the USA
```
Comment (cbc02009) [inline printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD:19]: same thing here.

File: printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+![Build Plate](./images/Dingushippo_plate.jpg)
+
+
+ **Links**
+
+[Protoloft- Self Calibrating Z Offset](https://github.com/protoloft/klipper_z_calibration)      
+
+[Zruncho3d - Zeroclick Z probe for V0.](https://github.com/zruncho3d/ZeroClick)      
+
+[HeartK1213 - Sexbolt Endstop](https://github.com/hartk1213/MISC/tree/main/Voron%20Mods/Voron%202/2.4/Voron2.4_SexBolt_ZEndstop)
```
Comment (cbc02009) [inline printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD:51]: I believe this was moved to a new repo (voron hardware maybe?). @hartk1213 can you confirm?

**Review by cbc02009 — CHANGES_REQUESTED** [review]

File: printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD (hunk 4 above)
Comment (hartk1213) [reply printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD:51]: also change it from Heartk1213 to Harkt1213

Comment (Polar-Ted) [reply printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD]: done

File: printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD (hunk 3 above)
Comment (Polar-Ted) [reply printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD]: done

File: printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD (hunk 2 above)
Comment (Polar-Ted) [reply printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD]: done

File: printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD (hunk 4 above)
Comment (Polar-Ted) [reply printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD]: Should I change to this link? https://github.com/VoronDesign/Voron-Hardware/tree/master/Microswitch_Z_Endstop

Comment (hartk1213) [reply printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD]: > Should I change to this link? https://github.com/VoronDesign/Voron-Hardware/tree/master/Microswitch_Z_Endstop I think the currently linked sexbolt endstop repository is more relevant to this assembly.

change it to this one 
https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/hartk1213/Voron2.4_SexBolt_ZEndstop

Comment (Polar-Ted) [reply printer_mods/Polar-Ted/V0.1_Its_Not_a_Sexbolt_switch/README.MD]: done

**Review by cbc02009 — APPROVED** [review]

**Pull request merged**